app/packs/src/fetchers/AdminFetcher.js

```diff
@@ -185,6 +185,15 @@ export default class AdminFetcher {
       .catch((errorMessage) => { console.log(errorMessage); });
   }
 
+  static fetchUsersByNameType(name, type, limit = 5) {
+    return fetch(`/api/v1/admin_user/listUsers/byname.json?type=${type}&name=${name}&limit=${limit}`, {
```

Comment:
Maybe use URLSearchParams for better clarity, but i think the current code is clear enough for me